### DISCUSSION TITLE
如何让MPV切换视频是保持窗口不变

### DISCUSSION THREAD

**Discussion by Lonely233233** [hooke007/mpv_PlayKit#289] [mpv 通用]
大佬请教一下怎么让怎么在切换视频时候保持窗口不变
我现在设置的geometry=60%这个
玩lol的时候看视频缩小了比列然后有些部分视频切换的时候会变回原来的geometry=60%
有什么方法可以保持比列不变吗

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/blob/175a07fdd33bad58d65edadf76e640549d0cebe6/portable_config/mpv.conf#L75

**Reply by Lonely233233** [reply]
大佬这个命令不能锁定窗口大小 有没有就是改变了大小就可能锁定的 有时玩游戏突然窗口就变大了

**Reply by hooke007** [reply]
我重看了一遍你的问题描述，你的意思是全屏游戏导致的系统分辨率变化？这会影响所有窗口大小，不仅仅是mpv

**Reply by Lonely233233** [reply]
可能是我表述有问题 我重新表述我现在的问题
现在我遇到的问题就是看视频的时候调整了窗口大小比如改成了1600x900
原来默认启动大小为1260*720改变了大小但是没有办法把大小锁死有些视频在下一集的时候会变回设置中的大小
没有办法一直锁定在1600x900
我想表达的意思有没有类似potplayer那样的改变窗口大小就保持住的命令然后换集窗口大小也不会改变

**Reply by hooke007** [reply]
我给你的选项就是你说的功能，不能实现就找log或控制台的消息查错

**Reply by Lonely233233** [reply]
谢谢大佬 我也是入坑mpv不是很懂

**Reply by Lonely233233** [reply]
我去控制台看看

**Reply by Lonely233233** [reply]
就部分集数 会导致mpv变回原来的大小
